lib/node_modules/@stdlib/blas/ext/base/zfill/lib/ndarray.js

```diff
@@ -128,7 +128,7 @@ function zfill( N, alpha, x, stride, offset ) {
 		j = ix * 2;
```

Comment:
We can save ourselves some flops here by refactoring to

```javascript
	strideX *= 2;
	ix *= 2;
	for ( i = 0; i < N; i++ ) {
		view[ ix ] = re;
		view[ ix+1 ] = im;
		ix += strideX;
	}
```

This also applies elsewhere. Namely, when we convert to a real-valued view at L87, we can also adjust the strides and offset accordingly. We can thus avoid needing to perform the `j = ix * 2` multiplication.